kbdd/src/test/kotlin/ru/fix/kbdd/rest/RestTest.kt

```diff
@@ -392,4 +393,31 @@ class RestTest {
                             """))
         )
     }
+
+    @Test
+    suspend fun `post multipart data`() {
+        request {
+            headers { "my-header" % "header-value" }
+            baseUri(server.baseUrl())
+            post("/post-form-data-request")
+            file("resource-with-placeholders.json", File("src/test/resources/resource-with-placeholders.json").inputStream())
+        }
+
+        statusCode().isEquals(200)
+        bodyJson()["status"].isEquals("success")
+
+        server.verify(
+                postRequestedFor(urlPathEqualTo("/post-form-data-request"))
+                        .withHeader("my-header", equalTo("header-value"))
+                        .withHeader("Content-Type", containing("multipart/form-data"))
+                        .withRequestBodyPart(
+                                aMultipart()
+                                        .withHeader("Content-Disposition", containing("form-data"))
+                                        .withHeader("Content-Disposition", containing("name=\"file\""))
+                                        .withHeader("Content-Disposition", containing("filename=\"resource-with-placeholders.json\""))
+                                        .withHeader("Content-Type", equalTo("application/octet-stream"))
```

Comment:
could you please add verification that actual content of the file was sent, like: 
`.withBody(containing("file content here"))`